lib/Doctrine/Common/Collections/ArrayCollection.php

```diff
@@ -325,12 +325,12 @@ public function getIterator() : Traversable
      * @return static
      *
      * @psalm-template U
-     * @psalm-param Closure(T=):U $func
+     * @psalm-param Closure(T=, TKey):U $func
```

Comment:
`TKey=`, since the parameter is optional

Reply:
:+1 done